### PR TITLE
Add new selinux plugin for Linux

### PR DESCRIPTION
**Pull request by davide125**
## Description
Add a basic plugin to expose SELinux status information. FYI, I have very little experience with SELinux and I'm definitely open for better ways to do this. We mostly need this for `current_mode` and `loaded_policy_name`, but I figured covering everything reported by `sestatus` could be useful.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Chore (non-breaking change that does not add functionality or fix an issue)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [ ] I have run the pre-merge tests locally and they pass.
- [ ] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [x] All commits have been signed-off for [the Developer Certificate of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### CODE REVIEW

File: lib/ohai/plugins/linux/selinux.rb [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+        unless key.start_with?("/")
+          key.downcase!
+          key.tr!(" ", "_")
+        end
+
+        selinux[section][key] = val
+      end
+    else
+      logger.trace("Plugin Selinux: Could not find sestatus. Skipping plugin.")
```
Comment (jaymzh) [inline lib/ohai/plugins/linux/selinux.rb:65]: Hey @lamont-granquist - should this be debug or trace? Feels like debug to me.

Comment (davide125) [reply lib/ohai/plugins/linux/selinux.rb:65]: I copied that from `lib/ohai/plugins/linux/sessions.rb` fwiw

Comment (tas50) [reply lib/ohai/plugins/linux/selinux.rb:65]: debug is probably fine 🤷

Comment (davide125) [reply lib/ohai/plugins/linux/selinux.rb]: Switched this to use debug, thanks @tas50

**Comment by tas50** [comment]
Thanks @davide125. This one is going to be handy. We might have to turn it on by default

**Pull request merged**